app/views/claims/claim_school.html.erb

```diff
@@ -3,27 +3,27 @@
 
     <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
-    <h1 class="govuk-heading-xl">
-      <%= t("tslr.questions.claim_school") %>
-    </h1>
 
-    <% if @schools.blank? %>
-
-      <% if params[:school_search].present? && current_claim.errors.empty? %>
-        <p class="govuk-body">
-          <strong>No results match that search term. Try again.</strong>
-        </p>
-      <% else %>
-        <span id="school-search-hint" class="govuk-hint">
-          Search for the school name, use at least four characters.
-        </span>
-      <% end %>
+      <h1 class="govuk-heading-xl">
+        <%= t("tslr.questions.claim_school") %>
+      </h1>
 
+    <% if @schools.blank? %>
+        <% if params[:school_search].present? && current_claim.errors.empty? %>
+          <p class="govuk-body">
+            <strong>No results match that search term. Try again.</strong>
+          </p>
+        <% else %>
+          <span id="school-search-hint" class="govuk-hint">
+            Search for the school name, use at least four characters.
+          </span>
+        <% end %>
       <%= form_tag(claim_path("claim-school"), method: :get) do %>
-        <div class="govuk-form-group">
-          <%= label_tag :school_search, "School name", class: "govuk-label" %>
-          <%= text_field_tag :school_search, params[:school_search], class: "govuk-input", value: current_claim.claim_school&.name %>
-        </div>
+          <%= form_group_tag current_claim, :school_search do %>
+            <%= label_tag :school_search, "School name", class: "govuk-label" %>
+            <%= errors_tag current_claim, :school_search %>
+            <%= text_field_tag :school_search, params[:school_search], id: :tslr_claim_school_search, class: css_classes_for_input(current_claim, :school_search), value: current_claim.claim_school&.name  %>
```

Comment:
For some reason the error message link doesn't move focus into the text field for the school search in the same way it does for the other text fields when there are errors. Any ideas?

Reply:
Great spot, it turned out to be the `id` of the label `for` not matching, I've declated both `id`s now and it is working.